lib/menu.ts

```diff
@@ -137,8 +137,11 @@ export class Menu extends Disposable implements IPopupContent {
       // Not using isSelectable because it checks the offset height of the elements to determine
       // visibility. None of the elements have an offset height on creation since they are not yet
       // attached to the dom.
-      const elems = Array.from(this.content.children).filter(elem => elem.hasAttribute('tabIndex'));
-      this._selected.set(elems[options.startIndex]);
+      const elems = Array.from(this.content.children).filter(elem =>
+        elem.hasAttribute('tabIndex') && !elem.hasAttribute('disabled'));
+      if (elems.length > options.startIndex) {
```

Comment:
How is this related to the commit ?

Reply:
The 'if' statement is only peripherally related - I figured I should add it to prevent an index error